platform-api/che-core-api-core/src/main/java/org/eclipse/che/api/core/model/project/type/ProjectType.java

```diff
@@ -25,7 +25,7 @@
 
     List<? extends Attribute> getAttributes();
 
-    List<? extends ProjectType> getParents();
+    List<ProjectType> getParents();
```

Comment:
Are you sure about this change? It seems that we don't have _DTO_ implemenation of `ProjecType`, but if we do it won't work in this way.